src/scss/components/_badges.scss

```diff
@@ -34,10 +34,6 @@ $badge-angled-rounded-radius: 2px !default;
         padding: $badge-padding;
         position: relative;
         white-space: nowrap;
-
-        span {
```

Comment:
Why do you need to remove the spacing? This will affect the menu page badges.

Reply:
yeah - i did put a thing in the description but its super missable. 😄 

Menu designers have agreed the changes (Pieter checked with Taimi / Bruno)

Reply:
OK, I'll update the HTML structure of my designs as I can see the issue in my layout now as well 📶